packages/grafana-ui/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -14,6 +14,9 @@ export interface ErrorBoundaryApi {
 }
 
 interface Props {
+  /** Unique identifier for the error boundary. Used when reporting errors in Faro. */
+  id?: string;
```

Comment:
I can see how a misuse of this prop might be harmful, such as accidentally putting a variable here. Can we either infer the ID instead, or enforce it with a lint rule?

Reply:
How important is the uniqueness? Will things break if it's not unique, or just be more confusing when looking through Faro data?

I don't think `id` is a good prop name for this, as I think this should be reserved for HTML ID attributes. Perhaps... `errorSource`? `boundryName`?

Reply:
>  just be more confusing when looking through Faro data?
This. The name is there so we can easily identify in which boundary the error happened.

`boundaryName` probably makes more sense.